src/Catty/FormulaEditor/ViewController/FormulaEditorViewController.m

```diff
@@ -272,13 +254,19 @@ - (void)viewDidLoad
     
     panRecognizer.delegate = self;
     
-    [self.variablePicker addGestureRecognizer:tapToSelect];
-    [self.variablePicker addGestureRecognizer:panRecognizer];
+    [self setupNavigationBar];
+    
+    self.formulaEditorSectionViewController = [[FormulaEditorSectionViewController alloc] init];
```

Comment:
Is the `FormulaEditorSectionViewController` also working without the `FormulaManager` and `SpriteObject`? If not, please add them directly add the initializer and make them non-optional, so that every instance must have those necessary properties assigned.